bundle/Command/MigrateDataV1ToV2Command.php

```diff
@@ -0,0 +1,239 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Netgen\Bundle\LayoutsRemoteMediaBundle\Command;
+
+use Doctrine\DBAL\Connection;
+use Doctrine\DBAL\Types\Types;
+use Symfony\Component\Console\Command\Command;
+use Symfony\Component\Console\Helper\ProgressBar;
+use Symfony\Component\Console\Input\InputInterface;
+use Symfony\Component\Console\Output\OutputInterface;
+
+use function array_shift;
+use function count;
+use function explode;
+use function implode;
+use function json_decode;
+use function json_encode;
+use function sprintf;
+
+use const PHP_EOL;
+
+class MigrateDataV1ToV2Command extends Command
```

Comment:
Missing final. Check other classes please :)

Reply:
@emodric Fixed in https://github.com/netgen-layouts/layouts-remote-media/pull/15/commits/177d05b82732914d2638b94093b66838f3fb7cf2. All other classes are already final except Doctrine entity.